UI/src/sass/option-select.scss

```diff
@@ -116,10 +116,19 @@ ul.gem-c-checkboxes__list {
   }
 }
 
+.app-c-option-select__heading {
+  display: none;
+}
+
 .js-enabled {
   .app-c-option-select__heading {
     position: relative;
     padding: 10px 8px 0px 35px;
+    display: block;
+  }
+
+  .app-c-option-select__legend {
+    display: none;
```

Comment:
Could we use the `govuk-visually-hidden` mixin here so that it's still available to screen readers?

Reply:
Good shout - I have updated this to extend the .govuk-visually-hidden class